### ISSUE TITLE
Support external scripts

### ISSUE THREAD

**Issue by JRJurman** [Tram-One/tram-lite#44]
## Summary

Potentially separate from `tl-effect` we should have some option for loading external scripts when a component mounts. This would be useful for cases where some external dependency has some library that it needs to include.

Some thinking needs to be done if this should be a `tl-effect`, or some other attribute (given that all the affordances for inline scripts wouldn't be available here).

**Comment by JRJurman** [comment]
That being said, it does occur to me that the same effect can be accomplished with the following:

```
<script tl-effect>
  fetch('your-external-script-here.js');
</script>
```

So, I'd be curious to learn and understand what use-cases this might not cover (granted, this is a deviation from what is probably an expected feature, so if people would just expect this to work, that's valid feedback too!)

**Comment by JRJurman** [comment]
This is now available as part of v5.2.0
